Tasks/js-targets.js

```diff
@@ -10,12 +10,7 @@ function compile(source, destination) {
     return gulp.src(source + '**/*.js')
         .pipe(eslint())
         .pipe(eslint.format())
-        .pipe(eslint.failOnError())
         .pipe(gulp.dest(destination));
 };
 
-function clean(destination) {
-    return async () => await del(destination + '**/*.js');
-}
-
-module.exports = { compile, clean };
```

Comment:
Ezt miért törölted ki?

Reply:
Sehol nem volt használva, ez egy leftover code volt [innen](https://github.com/Lombiq/Gulp-Extensions/blob/dev/Tasks/scss-targets.js)

Reply:
De nem jó, ha van a JS-hez is clean?

Reply:
Végül is, de, visszateszem

Reply:
Itt most ez van, szal értelmezésem szerint maga az ESLint failel:
![image](https://user-images.githubusercontent.com/1976647/108108533-06b49980-7091-11eb-8d61-c8d3e1ba62c2.png)

Reply:
Igen, ez itt és az scss-targets-nál van, de ötletem sincs miért, disable sem segít rajta

Reply:
Google sem mond semmit? ESLit bug report van róla?

Reply:
Szal?

Reply:
Ha itt: https://github.com/Lombiq/Gulp-Extensions/pull/11/files/ad3fe6ef70caac89a367df5f898996fcafed8524#diff-a9c2047e690de21a36b84317744d61c3757061892a8f2d04d3bffee314c6b8ab megnyitod, akkor tudsz válaszolni. Azt látom, hogy visszaraktad, viszont Google sem mond semmit? ESLit bug report van róla?

Reply:
"parserOptions": {
    "ecmaVersion": 6 -> 8
  }
  beállítás után, már rendesen mutatja, hogy 
  https://eslint.org/docs/rules/no-return-await#disallows-unnecessary-return-await-no-return-await
  ezt a rulet szegi meg

Reply:
Cool. @DAud-IcI @I3undy jó nekünk az ES 8?

Reply:
Örülök hogy IE-t dobtuk Coliban és mehet ES6, őszintén szólva fogalmam sincs.

Reply:
airbnb-ben is ecma version 6 van

Reply:
szóval ez ecma 8 eslint kijavítva failel mint gulp task úgyhogy inkább így hagyom

Reply:
Mármint kijavítva mire?

Reply:
https://github.com/Lombiq/Gulp-Extensions/blob/d5326d443fc081c68707d6b2b5893be77c4d0345/Tasks/js-targets.js#L16

Reply:
https://github.com/Lombiq/Gulp-Extensions/blob/d5326d443fc081c68707d6b2b5893be77c4d0345/Tasks/js-targets.js#L16-L19

Reply:
OK